src/main/java/io/github/railroad/welcome/WelcomeLeftPane.java

```diff
@@ -65,28 +67,28 @@ public RRListView<MenuType> getListView() {
     }
 
     public enum MenuType {
-        HOME("Home", FontAwesomeSolid.HOME),
-        NEW_PROJECT("New Project", FontAwesomeSolid.PLUS),
-        OPEN_PROJECT("Open Project", FontAwesomeSolid.FOLDER_OPEN),
-        IMPORT_PROJECT("Import Project", FontAwesomeSolid.FILE_IMPORT),
-        SETTINGS("Settings", FontAwesomeSolid.COG);
+        HOME("railroad.home.welcome.home", FontAwesomeSolid.HOME),
+        NEW_PROJECT("railroad.home.welcome.newproject", FontAwesomeSolid.PLUS),
+        OPEN_PROJECT("railroad.home.welcome.importproject", FontAwesomeSolid.FOLDER_OPEN),
+        IMPORT_PROJECT("railroad.home.welcome.importproject", FontAwesomeSolid.FILE_IMPORT),
+        SETTINGS("railroad.home.welcome.settings", FontAwesomeSolid.COG);
 
-        private final String name;
+        private final String key;
         private final Ikon icon;
         private final Paint color;
 
-        MenuType(String name, Ikon icon, Paint color) {
-            this.name = name;
+        MenuType(String key, Ikon icon, Paint color) {
+            this.key = key;
             this.icon = icon;
             this.color = color;
         }
 
-        MenuType(String name, Ikon icon) {
-            this(name, icon, Color.WHITE);
+        MenuType(String key, Ikon icon) {
+            this(key, icon, Color.WHITE);
         }
 
         public String getName() {
-            return this.name;
+            return this.key;
         }
```

Comment:
If this returns the key then the method should be called `getKey`. If it should still return the name then it should be localized.